launch/sim_iarc7_safety.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
Why don't we have it take an argument to a yaml file with the list? Anything else seems inconsistent and it means we can't add other settings to the safety yaml file if we need to.